governance-exercises/Le Bateleur - Interpretation Exercise 01/Scenarios.md

```diff
@@ -0,0 +1,45 @@
+
+**Poll 2** - "Governance Scope Bootstrapping Edits"
+
+"Signal your support or opposition to revising the Governance Scope to correct a typo preventing Aligned Delegates from being compensated." 
+
+Plurality poll; binary vote; and active for 3 days. 
+
+I would link to GOV 12 (as the authority for the poll); the Forum thread; and the Amendment Pull Request. Finally, I would link to the accompanying Poll #1.
+
+### Scenario B - A large token holder wants to add logic to the Stability Scope more quickly than is possible using the monthly governance cycle.
+
+No, I would not allow a vote to take place. No "unintended consequence or mistake in the Atlas and Scope Artifacts" is alleged or discernible. Even assuming this first-prong showing, there is no indication that a supposed mistake is *causing* Governance to function incorrectly, apart from the vague claim that the change is required to "prevent the DAO losing significant value". The naked desire to add logic more quickly than is possible using the monthly cycle is facially irrelevant to the determination of whether GOV 12 is applicable. 
+
+### Scenario C - An anonymous account wants to add logic to the Stability Scope more quickly than is possible using the monthly governance cycle.
+
+No, I would not allow a vote to take place. No "unintended consequence or mistake in the Atlas and Scope Artifacts" is alleged or discernible. Even assuming this first-prong showing, there is no indication that a supposed mistake is *causing* Governance to function incorrectly, apart from the vague claim that the change is required to "prevent the DAO losing significant value". The naked desire to add logic more quickly than is possible using the monthly cycle is facially irrelevant to the determination of whether GOV 12 is applicable. 
+
+### Scenario D - As a consequence of a significant reduction in Aligned Delegate pay, all Aligned Delegates step down from the position.
+
+No, I would not allow a vote to take place. By its explicit terms, [GOV 6.3](https://mips.makerdao.com/mips/details/MIP113#6-3-prime-delegate-and-reserve-delegate-slots) allows for Aligned Delegate compensation to be significantly reduced. It is unreasonable to suppose that the drafter did not foresee the contingency of Aligned Delegates reacting unfavorably to pay reductions, all the way up to resigning the position. In fact, the Atlas' [inclusion](https://mips.makerdao.com/mips/details/MIP101#2-6-3-1-1) of general restrictions and a one-month buffer period when adjusting Aligned Delegate compensation indicates the drafter accounted for this contingency of unfavorable reactions.
+
+In any event, no "unintended consequence or mistake in the Atlas and Scope Artifacts" is alleged or discernible. Absent this first-prong showing, it is not possible to meet the second-prong showing - namely, that such unintended consequence or mistake has *caused* Maker Governance to function incorrectly.
```

Comment:
Interesting. I can see why and how you've come to this conclusion, and I agree it follows from the wording. That said, I'm not sure it's the decision I would have made.

Reply:
Given the next paragraph, this makes more sense. Also, it is an interpretation exercise primarily, and I can't argue with the interpretation here.